easy_my_coop_website_portal/readme/DESCRIPTION.rst

```diff
@@ -3,4 +3,3 @@
 * On page /my/account
    * prevents modifications to "name" and "email"
    * makes "iban", "birthdate_date", "gender" and "lang" compulsory
-* ...
```

Comment:
@robinkeunen Please verify that this change is correct. pylint was complaining about this line, so I deleted it.

Reply:
👌